config/site_settings.yml

```diff
@@ -1481,6 +1481,9 @@ developer:
   enable_safe_mode:
     default: true
     client: true
+  refresh_maxmind_db_during_precompile_days:
+    default: 2
```

Comment:
We should probably add a range for that setting. Maybe 1-30 ?

Reply:
Maybe 0-120, if set to 0 it does not update.